### ISSUE TITLE
Better describe intention of `navigator.canShare()` and usage

### ISSUE THREAD

**Issue by saschanaz** [w3c/web-share#211]
Previous context: https://github.com/w3c/web-share/issues/108#issuecomment-697689527

```js
navigator.canShare({
  text: "foo",
  somethingUnsupported: {}
});
```

This should fail but currently it does not.

What weird is that `share()` will anyway succeed with that argument.

**Comment by saschanaz** [comment]
Hmmmmm.

Given that `share()` will succeed anyway, I wonder we can explicitly describe `canShare()` as "a check to ensure at least one of the fields are supported" and leave the behavior as-is. If all of them are unsupported then the current algorithm will ban it.

Wdyt? @marcoscaceres

**Comment by saschanaz** [comment]
It would also be worth mentioning that if support for a specific field is important, then the field should be passed alone for the expected result.

**Comment by marcoscaceres** [comment]
Definitely wouldn't hurt to provide such guidance. Do you want to have a go at putting together some wording for it or suggest something here?

**Comment by saschanaz** [comment]
I doubt I'll have time to do that too soon, but something like:

> NOTE: canShare() only checks the validity of the supported values and ignores the unsupported ones.
> 
> ```js
> const result = navigator.canShare({
>   text: "Star Wars",
>   darthVader: window.parent // this is completely ignored
> });
> result // true, since `text` can be shared
> ```
> 
> To check whether a specific field is supported, pass that field as the sole one.
> 
> ```js
> const result = navigator.canShare({
>   darthVader: window.parent
> });
> result // false
> ```
> 
> Unsupported values being ignored will make the object be perceived as an empty dictionary, which will be rejected by canShare() as nothing can be shared.

**Comment by marcoscaceres** [comment]
That’s great! Thank you! I’ll adapt that and ping you for review.

**Comment by marcoscaceres** [comment]
Closed by #211
